cumulus_library/cli_parser.py

```diff
@@ -124,8 +121,15 @@ def get_parser() -> argparse.ArgumentParser:
     clean = actions.add_parser(
         "clean", help="Removes tables & views beginning with '[target]__' from Athena"
     )
+
     add_target_argument(clean)
+    add_verbose_argument(clean)
     add_aws_config(clean)
+    clean.add_argument(
+        "--explicit_prefix",
```

Comment:
A) hyphen not underscore my dude

B) I know why you named it this - in your head you were thinking "this is the explicit string I want to check against".

But my head bumped against this a little bit, feeling like the intent should actually be reversed. With this flag enabled, it's actually a really loosey goosey prefix. Potentially matching multiple studies. Without this flag, you have a tightly defined explicit one-study prefix.

Would users of the CLI even really think of it in terms of a "prefix?" Like, to me I think of it as "I'm passing a study name, and the tool is managing how it cleans that up." Internally, we often use it as a prefix. But that's not what I would be thinking of when I pass in the CLI arg.

I don't have a better arg name though. `--wildcard-target`... `--regex-target` (and actually allow a regex?)

I dunno. Maybe the possible confusion here is light and your arg is perfect. Will defer to your brain on this.

Reply:
A) i flipped it, and then went back on it, because i've already committed the sin here elsewhere. i should make a ticket to clean it up.

B) this is definetely a 'i am thinking about this based on the way the non-flag version of this works' - i.e. you infer the prefix from the study name in the manifest, and then add a dunder to it, so you always have a coerced prefix. When you're explicitly saying 'use this prefix i'm giving you and trust me on it and don't bother checking a manifest' with this flag.

Does that help? I don't love the name either, open to suggestions.

Reply:
B) Yeah I guess I was bumping on the fact that this name is referencing some internal details instead of abstracting it. Like, in Matt's brain, deeply steeped in this code, obviously the `--target` is a prefix that we append a dunder to. But a poor little user might just be thinking "pls delete this study's tables, I don't care or know how that works behind the scenes". The fact that it is internally a prefix may not occur to them, so `--explicit-prefix` might not register as anything. That's all I was getting at.

Plus to my external brain, I was thinking that the default behavior actually uses a more explicit prefix than _with_ the flag. So even for a half-steeped person, this was confusing.

That all said, :shrug: you can just document it well enough and it's fine (this is a high risk place to have some confusion, but casuals hopefully aren't running this or adding arguments they don't understand).

Reply:
`--specify-prefix`? and then a big warning comment about how 99% of the time you shouldn't need to use it?

Reply:
I think I like `--explicit-prefix` better, despite my whinging. Docs can make it clear. Though... what about just `--prefix`? If you don't like that, I'd say stick with `--explicit-prefix`

Reply:
`--prefix` it is